### ISSUE TITLE
Accessibility of user help pages

### ISSUE THREAD

**Issue by PaulMorris** [SolutionGuidance/psm#700]
There's an accessibility issue on the user help pages.  The search form is missing a label/title. Possible fixes for that would be an upstream contribution to Sphinx or a local modification of the Sphinx theme we are using.
